### ISSUE TITLE
request an example of openid intergration

### ISSUE THREAD

**Issue by jor112358** [lepture/authlib#97]
can I replace flask-openid with authlib?

if the answer is 'yes', could you pls show an example?

many thx!

**Comment by lepture** [comment]
No. Authlib only has openid connect for now. OpenID Connect is different than OpenID.
